pymatgen/io/vasp/MatPESStaticSet.yaml

```diff
@@ -0,0 +1,160 @@
+# Default VASP settings for static calculations in the MatPES Project
+# using the Regularized-Restored Strongly Constrained and Appropriately
+# Normed functional (PBE, PBE+U or r2SCAN).
+INCAR:
+  ALGO: ALL
+  EDIFF: 1.e-05
+  EDIFFG: -0.02
+  ENAUG: 1360
+  ENCUT: 680
+  IBRION: 2
+  ISIF: 3
+  ISMEAR: 0 # change from 2 to 0, included to have some reasonable default
+  ISPIN: 2
+  KSPACING: 0.22 # included to have some reasonable default
+  LAECHG: True
+  LASPH: True
+  LCHARG: True
+  LELF: False # LELF = True restricts calculation to KPAR = 1
+  LMIXTAU: True
+  LORBIT: 11
+  LREAL: Auto
+  LVTOT: True
+  LWAVE: False
+  METAGGA: PBE # change R2SCAN to PBE as default
+  NELM: 200
```

Comment:
We may possibly want to increase this.

Reply:
Should I keep the value of NELM:200 for now?

Reply:
This is fine.